marklogic-client-api/src/main/java/com/marklogic/client/type/PlanSearchOptions.java

```diff
@@ -15,19 +15,19 @@
  */
 package com.marklogic.client.type;
 
-// IMPORTANT: Do not edit. This file is generated.
-
 /**
  * An option controlling the scoring and weighting of fromSearch()
  * for a row pipeline.
  */
 public interface PlanSearchOptions {
-    XsDoubleVal getQualityWeight();
+	XsFloatVal getQualityWeight();
```

Comment:
Given that this is changing due to a bug and it's a public API, add the following javadoc for it:

```
/**
 * Changed in release 6.7.0 to return a float, as the server requires a float and throws an error on a double. 
 */ 
```